atomic

```diff
@@ -302,18 +303,35 @@ def create_parser(atomic):
                                  "install method"))
 
     # atomic images
-    imagesp = subparser.add_parser(
-        "images", help=_("list container images on your system"),
-        epilog="atomic images by default will list all installed "
-        "container images on your system.  Using the --prune "
-        "option, will free up disk space deleting unused "
-        "'dangling' images")
-    imagesp.set_defaults(func='images')
-    imagesp.add_argument("--prune", action="store_true",
+    imagesp = subparser.add_parser("images")
+    images_subparser = imagesp.add_subparsers(title='images subcommands',
+                                              description="operate on images",
+                                              help='additional help')
+    # atomic images list
+    list_parser = images_subparser.add_parser("list",
+                                              help=_("list container images on your system"),
+                                              epilog="atomic images by default will list all installed "
+                                                     "container images on your system.  Using the --prune "
+                                                     "option, will free up disk space deleting unused "
+                                                     "'dangling' images")
+    list_parser.set_defaults(func='images')
+    list_parser.add_argument("--prune", action="store_true",
                          help=_("prune dangling images"))
-    imagesp.add_argument("-n", "--noheading", dest="heading", default=True,
-                         action="store_false",
-                         help=_("do not print heading when listing the images"))
+
+    list_parser.add_argument("-n", "--noheading", dest="heading", default=True,
+                     action="store_false",
+                     help=_("do not print heading when listing the images"))
+
+    # atomic images delete
+    delete_parser = images_subparser.add_parser("delete",
+                                                help=_("mark image for deletion"),
+                                                epilog="Marks image. registry garbage-collection "
+                                                "when invoked will recover used disk space")
+    delete_parser.set_defaults(_class=Delete, func='delete_image')
```

Comment:
I would think this would delete the local image not the image at the registry?  
What do you think of ?
```
atomic image delete IMAGE

atomic image delete --remote IMAGE
``